src/main/java/org/apache/sysml/parser/dml/DmlSyntacticValidator.java

```diff
@@ -519,7 +532,8 @@ public void exitBuiltinFunctionExpression(BuiltinFunctionExpressionContext ctx)
 		String functionName = names[1];
 
 		ArrayList<ParameterExpression> paramExpression = getParameterExpressionList(ctx.paramExprs);
-
+		//convert the namespace for the function identifier arguments
+		convertNamespace(paramExpression);
```

Comment:
Any modification to the syntactic parser for DML would also need to go into the syntactic parser for PyDML. Hence, if possible, it would be good to leave the syntactic parsers untouched and move any preprocessing into the semantic parsing, i.e., parse tree validation.